### PR TITLE
Remove all non ascii chars

### PR DESCRIPTION
**Pull request by collivier**
é in spécial is allowed in chapter01.rst

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

**Review by CsatariGergely — APPROVED** [review]

**Comment by CsatariGergely** [comment]
Uh-oh. Linkcheck for  http://testresults.opnfv.org/test/api/v1/results fails. Is this the same issue what ONAP has?

File: doc/ref_arch/openstack/chapters/chapter03.rst [hunk 1]
```diff
@@ -324,7 +324,7 @@ used:
 -  Foundation node
 -  Control nodes
 -  Compute nodes
--  Other supporting service nodes, e.g. network, shared storage, logging,
+-  Other supporting service nodes, e.g. network, shared storage, logging,
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter03.rst:327]:
```suggestion
-  Other supporting service nodes, e.g., network, shared storage, logging,
```

File: doc/ref_arch/openstack/chapters/chapter03.rst [hunk 2]
```diff
@@ -481,7 +481,7 @@ Manager in the Reference Model (RM) :cite:p:`refmodel`.
 
 All components must be deployed within a high available architecture
 that can withstand at least a single node failure and respects the
-anti-affinity rules for the location of the services (i.e. instances of
+anti-affinity rules for the location of the services (i.e. instances of
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter03.rst:484]:
```suggestion
anti-affinity rules for the location of the services (i.e., instances of
```

File: doc/ref_arch/openstack/chapters/chapter03.rst [hunk 3]
```diff
@@ -596,7 +596,7 @@ Virtualisation is a technology that enables a guest Operating System
 (OS) to be abstracted from the underlying hardware and software. This
 allows to run multiple Virtual Machines(VMs) on the same hardware. Each
 such VMs have their own OS and are isolated from each other
-i.e. application running on one VM does not have the access to resources
+i.e. application running on one VM does not have the access to resources
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter03.rst:599]:
```suggestion
i.e., an application running on one VM does not have access to the resources
```

File: doc/ref_arch/openstack/chapters/chapter03.rst [hunk 4]
```diff
@@ -702,7 +702,7 @@ Cloud Topology
 
 A telco cloud will typically be deployed in multiple locations ("sites")
 of varying size and capabilities (HVAC, for example); or looking at this
-in the context of OpenStack, multiple clouds (i.e. OpenStack end-points)
+in the context of OpenStack, multiple clouds (i.e. OpenStack end-points)
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter03.rst:705]:
```suggestion
in the context of OpenStack, multiple clouds (i.e., OpenStack end-points)
```

File: doc/ref_arch/openstack/chapters/chapter06.rst [hunk 5]
```diff
@@ -561,7 +561,7 @@ applied:
    protected
 -  Integrity of the deployment images must be checked, before starting
    deployment
--  Deployment must be done through a dedicated network (e.g. VLAN)
+-  Deployment must be done through a dedicated network (e.g. VLAN)
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter06.rst:564]:
```suggestion
-  Deployment must be done through a dedicated network (e.g., VLAN)
```

File: doc/ref_arch/openstack/chapters/chapter06.rst [hunk 6]
```diff
@@ -691,7 +691,7 @@ Where to Log
    manner where the event can be linked to the resource on which it
    occurred.
 -  Where technically feasible, events must be recorded on the device
-   (e.g. VM, physical node, etc.) where the event occurs, if the
+   (e.g. VM, physical node, etc.) where the event occurs, if the
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter06.rst:694]:
```suggestion
   (e.g., VM, physical node, etc.) where the event occurs, if the
```

File: doc/ref_cert/RC2/chapters/chapter02.rst [hunk 7]
```diff
@@ -470,7 +470,7 @@ tagged as high by
 -  PrivilegeEscalation
 
 At the time of writing, none of the Center for Internet Security (CIS)
-rules are defined as mandatory (e.g. sec.std.001: The Cloud Operator
+rules are defined as mandatory (e.g. sec.std.001: The Cloud Operator
```
Comment (pgoyal01) [inline doc/ref_cert/RC2/chapters/chapter02.rst:473]:
```suggestion
rules are defined as mandatory (e.g., sec.std.001: The Cloud Operator
```

**Review by pgoyal01 — CHANGES_REQUESTED** [review]

**Review by steelescotr — APPROVED** [review]

**Review by pgoyal01 — APPROVED** [review]

**Comment by collivier** [comment]
testresults.opnfv.org is down. it must be out of space as usual. I will clean it later today. Yes ONAP has its test database there too.

**Review by karinesevilla — APPROVED** [review]

**Pull request merged**